requirements/edx/github.txt

```diff
@@ -89,6 +89,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.5#egg=xblock-utils==1.0.5
 git+https://github.com/edx/edx-user-state-client.git@1.0.1#egg=edx-user-state-client==1.0.1
 git+https://github.com/edx/xblock-lti-consumer.git@v1.1.2#egg=lti_consumer-xblock==1.1.2
 git+https://github.com/edx/edx-proctoring.git@0.18.0#egg=edx-proctoring==0.18.0
+git+https://github.com/edx/help-tokens.git@nedbat/help-tokens-v1#egg=help-tokens==0.0
```

Comment:
This will change to a PyPI reference once the review of help-tokens (https://github.com/edx/help-tokens/pull/1) is done and the code pushed to PyPI.